exporter/syslogexporter/README.md

```diff
@@ -0,0 +1,111 @@
+# Syslog Exporter
+
+**Stability level**: Alpha
+
+## About The Exporter
```

Comment:
Do we need this header? Maybe just drop it? I think the other components don't have the "About" section specifically called out.

Reply:
changed